source/application/AMPIC887DriverServer/AMConsoleInputHandler.cpp

```diff
@@ -0,0 +1,64 @@
+#include "AMConsoleInputHandler.h"
+#include "AMAbstractConsoleCommandParser.h"
+
+AMConsoleInputHandler::AMConsoleInputHandler(AMAbstractConsoleCommandParser* commandParser,
+										   QObject* parent)
+	:QObject(parent),
+	  standardOutputStream_(stdout),
```

Comment:
I'm not the biggest fan of initializing member variables in the constructor declaration like this.  Is there a particular reason for doing it this way?  I can also just suck it up if it provides some other inherent benefit (e.g. more efficient, etc.)